src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -297,12 +302,12 @@ const ReleaseContent = () => {
             {allMethodologies.length > 0 && (
               <>
                 <h3
-                  className="govuk-heading-s govuk-!-margin-top-6"
+                  className="govuk-heading-sgovuk-!-padding-top-0"
```

Comment:
Is this classname correct? (govuk-heading-sgovuk-!-padding-top-0). Just wondering if it should be `govuk-heading-s govuk-!-padding-top-0` instead?